### ISSUE TITLE
cannot found git on windowns 10

### ISSUE THREAD

**Issue by litefeel** [desktop/desktop#2115]
![qq 20170625160012](https://user-images.githubusercontent.com/1288575/27514511-63854a02-59bf-11e7-8f31-ef94e40bb73f.jpg)

### Description
I installed git command from git-scm.com, but it talk me can not found git.

### Version

**GitHub Desktop version:** 0.6.2

**OS version:** [windows 10.0.15063]

### Steps to Reproduce

1. Clone any git repository
1. Click menu `Repository`-> `Open Command Prompt`
1. The message that displays the picture above

**Expected behaviour:** Cannot found git

**Actual behaviour:** Open git command line

**Reproduces how often:** Always

**Comment by Solen-Win** [comment]
Read and edits.

**Comment by litefeel** [comment]
![qq 20170625173441](https://user-images.githubusercontent.com/1288575/27515014-90de7d22-59cc-11e7-94b1-1576cbf9e19a.jpg)
I want to open this app.

**Comment by shiftkey** [comment]
@litefeel that dialog is a message we display when we can't find Git on your `PATH` - we could disregard this and continue to show the command prompt, but then any Git commands would fail.

Do you have Git for Windows installed? If so, could you tell me about your setup as maybe we're not detecting it correctly. If not, the `Install Git` button on that dialog should take you to [this documentation](https://help.github.com/articles/set-up-git/#setting-up-git) which then links off to the download link you need.

**Comment by litefeel** [comment]
@shiftkey I installed git on windows. but I have not put git to path.  i want to open MINGW64 or power shell, because they has color.

**Comment by shiftkey** [comment]
> I installed git on windows. but I have not put git to path.

You'll need to do this for Git to work from vanilla `cmd` as it currently works in Desktop.

> i want to open MINGW64 or power shell, because they has color.

This is a separate issue, see #888 for details.

**Comment by shiftkey** [comment]
As we defer to what's registered on `PATH` rather than anything more complex, I think this is all working as expected.

**Comment by cScarlson** [comment]
Windows 10
`$ git --version` --> `git version 2.15.1.windows.2`.
Though, I cannot find GitBash using `Win + "git"`. This occurred after updating to the version above. When I clicked on the GitBash shortcut icon on my desktop, it muttered something about "file doesn't exist in the location for this shortcut... would you like to delete this shortcut? Yes|No".

**Comment by shiftkey** [comment]
@cScarlson 

> When I clicked on the GitBash shortcut icon on my desktop, it muttered something about "file doesn't exist in the location for this shortcut... would you like to delete this shortcut? Yes|No".

This feels like it's an installation issue related to [Git for Windows](https://git-for-windows.github.io/) rather than Desktop. If you can reproduce the issue please submit an issue to [their issue tracker](https://github.com/git-for-windows/git).

**Comment by cScarlson** [comment]
@shiftkey, I just resolved it. I did the magical turn it off and then back on again. Not sure if I can reproduce it now or if I'll have time to for a while, but here's the workaround:

I uninstalled Git-2.15.1-64-bit, reinstalled, and accepted all defaults.

I feel its likely to relate to Git for Windows -- perhaps, even between the version I *was* using and the aforementioned. I'm not very good at keeping git updated :-P
